extensions/amp-access/0.1/amp-access.js

```diff
@@ -599,10 +619,16 @@ export class AccessService {
   /**
    * The promise will be resolved when a view of this document has occurred. It
    * will be rejected if the current impression should not be counted as a view.
+   * @param {time} timeToView
    * @return {!Promise}
    * @private
    */
-  whenViewed_() {
+  whenViewed_(timeToView) {
+    if (timeToView == 0) {
```

Comment:
Documented. The reason for 0 is when this method is called as the result of the user action.